function_calling/README.md

```diff
@@ -96,7 +122,24 @@ We provide a simple module for using the Function Calling LLM, for this you will
 
     we provide an [usage notebook](notebooks/usage.ipynb) that you can use as a guide for using the function calling module
 
-# Customizing the Funtion Calling module
+
+## Streamlit App
+
+We provide a simple GUI that allows you to interact with your function calling model
+
+To run it execute the following command 
+
+```bash
+    streamlit run streamlit/app.py --browser.gatherUsageStats false 
+```
+
+After deploying the starter kit GUI App you see the following user interface:
+
+![capture of function calling streamlit application](./docs/function_calling_app.png)
+
+there you will be able to select which tools you want the model to have access to and the max number of iterations you want the model to run to solve your query.
```

Comment:
On that page you will be able to select your function calling tools and the max number of iterations available for the model to answer your query.

Reply:
added